src/core/lib/experiments/experiments.yaml

```diff
@@ -228,3 +228,9 @@
   expiry: 2024/01/01
   owner: ctiller@google.com
   test_tags: []
+- name: block_excessive_requests_before_settings_ack
```

Comment:
Does this need an entry in rollouts.yaml?

Reply:
done